Sources/Sentry/Public/SentryScope.h

```diff
@@ -17,6 +17,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString *const SENTRY_CONTEXT_OS_KEY = @"os";
+static NSString *const SENTRY_CONTEXT_DEVICE_KEY = @"device";
```

Comment:
`h`: What drove you to make these public? I would prefer not to, if we don't have to.

Reply:
Needed them to be accessible in ObjC and Swift, but you are right. Will move to the private Swift Header.